docs/idle-crafting.rst

```diff
@@ -0,0 +1,56 @@
+idle-crafting
+=============
+
+.. dfhack-tool::
+    :summary: Allow dwarfs to automatically satisfy their need to craft objects.
+    :tags: fort gameplay
+
+This script allows dwarves to automatically satisfy their crafting needs. The
+script is configured through an overlay that is added to the main page of
+craftsdwarf's workshops, as described below.
+
+Usage
+-----
+
+::
+
+    idle-crafting status
+    idle-crafting disable
+    disable idle-crafting
```

Comment:
I suggest choosing one to be the canonical form, and only documenting that form. Both can be supported by the code, though, for flexibility.